runtime/tests/integration/helpers.rs

```diff
@@ -20,11 +23,13 @@ use tonic::{
     transport::{Channel, Server},
     Request, Response, Status,
 };
+use ulid::Ulid;
 
 pub struct TestRuntime {
     pub runtime_client: RuntimeClient<ClaimService<InjectPropagation<Channel>>>,
     pub bin_path: String,
     pub service_name: String,
+    pub deployment_id: Ulid,
```

Comment:
nit: Not sure if this is intentional, but we currently use `UUIDv4` for deployment ids. We've migrated only the service ids to `ulid`s, for the time being (we'll probably do this for deployments with the advent of the new deployer). For testing purposes I don't think it matters what we use since the runtime is agnostic (it accepts a `String`), but it just misleads/adds confusion if it doesn't correspond with the upstream data type.

Reply:
Ah, yes, good catch! This is probably since the new deployer used ulid for deployment ID, I will switch it back to uuid.

Reply:
Or switch it to `String` to make the future change easier

Reply:
This was the case in a couple of other places too, addressed in [de8c2c1](https://github.com/shuttle-hq/shuttle/pull/1139/commits/de8c2c16f1d149ad7679cf0f9a8bbe67b42b7b8b)

Reply:
I switched it to String in the logger in [53ab231](https://github.com/shuttle-hq/shuttle/pull/1139/commits/53ab2312540bc350ab4a70023733a269fcbeb25d)